### PR TITLE
Added metadata for docs building from doc-sle.

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Adding metadata for docs built from this repository.

### CODE REVIEW

File: xml/art_admin_slemicro_podman.xml [hunk 1]
```diff
@@ -7,11 +7,36 @@
...
-<?dbtimestamp format="B d, Y"?></date>
+    <productnumber>&productnumber;</productnumber>
+    <productname>&productname;</productname>
+    <productname role="abbrev">&productnameshort;</productname>
+    <date>
+<?dbtimestamp format="B d, Y"?
+>
+</date>
+<meta name="title" its:translate="yes">Podman Guide</meta>
+<meta name="description" its:translate="yes">Basic operations on containers using Podman.</meta>
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_podman.xml:21]:
```suggestion
<meta name="description" its:translate="yes">Basic operations on containers using Podman</meta>
```

File: xml/art_admin_slemicro_podman.xml [hunk 2]
```diff
@@ -7,11 +7,36 @@
...
+<meta name="social-descr" its:translate="yes">Using Podman to manage containers.</meta>
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_podman.xml:22]:
```suggestion
<meta name="social-descr" its:translate="yes">Using Podman to manage containers</meta>
```

File: xml/book_autoyast.xml [hunk 3]
```diff
@@ -20,8 +21,31 @@
...
   </dm:docmanager>
   <title>&ayguide;</title><productname>&productname;</productname>
-  <productnumber>&productnumber;</productnumber><date>
+  <productnumber>&productnumber;</productnumber>
+  <productname role="abbrev">&productnameshort;</productname>
+  <date>
 <?dbtimestamp format="B d, Y" ?></date>
+<meta name="title" its:translate="yes">&productnameshort; &ay; Guide</meta>
+    <meta name="description" its:translate="yes">Creating an &ay; profile to automaticaly deploy &productnameshort;.</meta>
+    <meta name="social-descr" its:translate="yes">Using &ay; to automaticaly deploy &productnameshort;.</meta>
```
Comment (dariavladykina) [inline xml/book_autoyast.xml:30]:
```suggestion
    <meta name="description" its:translate="yes">Creating an &ay; profile to automatically deploy &productnameshort;</meta>
    <meta name="social-descr" its:translate="yes">Using &ay; to automatically deploy &productnameshort;</meta>
```

File: xml/book_upgrade_slemicro.xml [hunk 4]
```diff
@@ -10,12 +10,38 @@
...
-  <productnumber>&productnumber;</productnumber><date>
+  <productnumber>&productnumber;</productnumber>
+  <productname role="abbrev">&productnameshort;</productname>
+  <date>
 <?dbtimestamp format="B d, Y"?></date>
+<meta name="title" its:translate="yes">&productnameshort; Upgrade Guide</meta>
+    <meta name="description" its:translate="yes">Upgrade of &productnameshort; from previous
+    releases using the &tr-up; command.</meta>
+    <meta name="social-descr" its:translate="yes">Upgrade of &productnameshort; from previous
+      releases.</meta>
```
Comment (dariavladykina) [inline xml/book_upgrade_slemicro.xml:26]:
```suggestion
    <meta name="description" its:translate="yes">Upgrade of &productnameshort; from previous
    releases using the &tr-up; command</meta>
    <meta name="social-descr" its:translate="yes">Upgrade of &productnameshort; from previous
      releases</meta>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see suggestions from my side. Generally we don't use full stops in titles and meta titles+meta descriptions etc., for that matter.

**Review by janajaeger — COMMENTED** [review]
Not sure whether "Initial version" merits a . at the end. That's one for Daria. Otherwise LGTM.

**Comment by dariavladykina** [comment]
Good point. I tend to not have the full stop. I think this must be in the template for future use (if it's not there yet).

File: xml/art_admin_slemicro_podman.xml [hunk 5]
```diff
@@ -7,11 +7,35 @@
...
+<meta name="title" its:translate="yes">Podman Guide</meta>
+<meta name="description" its:translate="yes">Basic operations on containers using Podman</meta>
+<meta name="social-descr" its:translate="yes">Using Podman to manage containers</meta>
+
+<revhistory xml:id="rh-article-podman">
+  <revision>
+    <date>2023-10-26</date>
+    <revdescription>
+      <para>
+        Initial version.
```
Comment (dariavladykina) [inline xml/art_admin_slemicro_podman.xml:29]:
```suggestion
        Initial version
```

File: xml/book_deployment_slemicro.xml [hunk 6]
```diff
@@ -13,12 +13,32 @@
...
   <productname>&productname;</productname>
   <productnumber>&productnumber;</productnumber>
+  <productname role="abbrev">&productnameshort;</productname>
   <date><?dbtimestamp format="B d, Y"?></date>
+  <revhistory xml:id="rh-book-deployment-slemicro">
+    <revision>
+      <date>2023-10-26</date>
+      <revdescription>
+        <para>
+          Initial version.
```
Comment (dariavladykina) [inline xml/book_deployment_slemicro.xml:29]:
```suggestion
          Initial version
```

File: xml/book_upgrade_slemicro.xml [hunk 7]
```diff
@@ -10,12 +10,38 @@
...
+    releases using the &tr-up; command</meta>
+    <meta name="social-descr" its:translate="yes">Upgrade of &productnameshort; from previous
+      releases</meta>
+    
+    <revhistory xml:id="rh-book-upgrade-slemicro">
+      <revision>
+        <date>2023-10-26</date>
+        <revdescription>
+          <para>
+            Initial version.
```
Comment (dariavladykina) [inline xml/book_upgrade_slemicro.xml:33]:
```suggestion
            Initial version
```

**Pull request merged**